Makefile

```diff
@@ -14,6 +14,6 @@
 all: test-integration
 
 test-integration:
-	go test ./... -args -integration -cluster-location=$(GCP_REGION) -cluster-name=$(KUBE_CLUSTER) -namespace-name=$(KUBE_NAMESPACE) -project-id=$(GCP_PROJECT)
+	go test -mod=vendor ./... -args -integration -cluster-location=$(GCP_REGION) -cluster-name=$(KUBE_CLUSTER) -namespace-name=$(KUBE_NAMESPACE) -project-id=$(GCP_PROJECT)
```

Comment:
Looks like the next version of Go won't need this (https://github.com/golang/go/issues/33848)... Up to you if it's worth mentioning in a comment.

Reply:
Not really, we use this in the sidecar repo, we just missed it here.